components/bl-rating-component/src/index.js

```diff
@@ -0,0 +1,95 @@
+import { useCallback, useRef, useState } from 'react';
+
+import { useRatingClassList } from './hooks/useRatingClassList';
+import { iconParams } from './utils/icon-params';
+import { iconsList } from './utils/icons-list.js';
+import { roundValueToPrecision } from './utils/number';
+import { RatingIcon } from './rating-icon';
+
+const RATING_PRECISION = 0.5;
+
+export default function RatingComponent({
```

Comment:
in one line 

`export default function RatingComponent({ component, eventHandlers }) {`